3gx/PokeReader.plgInfo

```diff
@@ -2,8 +2,8 @@ Author: Zaksabeast
 
 Version: # Plugin version
     Major: 1
-    Minor: 0
-    Revision: 0
+    Minor: 8
+    Revision: 4
```

Comment:
I think these are fine to remain as `0`. I'm not sure they're used anywhere, and the display only uses the values in `Cargo.toml`.